nix/flakebox.nix

```diff
@@ -652,17 +645,48 @@ rec {
 
       gatewayd = pkgs.dockerTools.buildLayeredImage {
         name = "gatewayd";
-        contents = [ gateway-pkgs pkgs.bash pkgs.coreutils pkgs.sqlite pkgs.fakeNss pkgs.busybox pkgs.curl pkgs.rsync ];
+        contents = [ gateway-pkgs ] ++ defaultPackages;
         config = {
           Cmd = [
             "${gateway-pkgs}/bin/gatewayd"
           ];
         };
       };
 
+      cln-light-gateway = let
+        entrypoint = pkgs.writeShellScriptBin "entrypoint.sh" ''
+          ${pkgs.clightning}/bin/lightningd \
+            --lightning-dir=/lightning \
+            --disable-plugin=bcli \
+            --network=$NETWORK \
+            --plugin=${pkgs.trustedcoin}/bin/trustedcoin \
+            --plugin=${gateway-pkgs}/bin/gateway-cln-extension \
+            --fm-gateway-listen=127.0.0.1:3301 \
```

Comment:
This needs to be 0.0.0.0:3301 for the docker port mapping to work for a docker compose, tested it with fedi devops guys and if you do it with 127.0.0.1 you can't talk across services to it like http://cln:3301 . See comment in main thread for details

Reply:
Great find, thx, makes sense! Will change

Reply:
just tweaked it, will land